packages/dashboard/src/components/Wizard/Wizard.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { IWizardStepProps, IWizardProps, IWizardStyles } from './Wizard.types';
+import { SubwayNav } from '../SubwayNav/SubwayNav';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { getWizardStyles } from './Wizard.styles';
+import { SubwayNavStepState, ISubwayNavStep } from '../SubwayNav/SubwayNav.types';
+
+/** Component for Wizard */
+export class Wizard extends React.Component<IWizardProps, {}> {
+  constructor(props: IWizardProps) {
+    super(props);
+  }
+
+  public render(): React.ReactNode {
+    const { steps } = this.props;
+
+    if (steps.length === 0) {
+      throw new Error('Wizard must have atleast one step.');
+    }
+
+    const navSteps = steps.map((step: IWizardStepProps, index: number) => {
+      const navStep: ISubwayNavStep = {
+        key: step.key,
+        label: step.label,
+        state: step.state,
+        disabled: step.disabled,
+        onClickStep: step.onClickStep,
+        isSubStep: step.isSubStep,
+        subSteps: step.subSteps
+      };
+
+      if (index > 0 && this.props.allowSkipAhead === false && navStep.state === SubwayNavStepState.NotStarted) {
+        // If allowSkipAhead is not allowed, then disable all steps that are "NotStarted"
+        // Except for first step, it cannot be disabled.
+        navStep.disabled = true;
+      }
+      return navStep;
+    });
+
+    const getClassNames = classNamesFunction<IWizardProps, IWizardStyles>();
+    const classNames = getClassNames(getWizardStyles!);
+
+    const stepContentToShow = this._getStepContentToShow();
+
+    const wizardStepProps: IWizardStepProps = {
+      key: stepContentToShow.key,
+      label: stepContentToShow.label,
+      state: stepContentToShow.state,
+      disabled: stepContentToShow.disabled,
+      onClickStep: () => {
+        stepContentToShow!.onClickStep;
+      },
+      wizardContent: stepContentToShow.wizardContent,
+      subSteps: stepContentToShow.subSteps
+    };
+
+    if (stepContentToShow.wizardContent === undefined) {
+      throw new Error('Content missing in the step - ' + stepContentToShow.key);
+    }
+
+    return (
+      <div className={classNames.wizardContentNavContainer}>
+        <div className={classNames.subwayNavSection}>
+          <SubwayNav steps={navSteps} wizardComplete={this.props.wizardComplete} />
+        </div>
+        <div className={classNames.contentSection}>
+          <div>{wizardStepProps.wizardContent!.contentTitle}</div>
+          <div className={classNames.content}>{wizardStepProps.wizardContent!.content}</div>
+        </div>
+      </div>
+    );
+  }
+
+  // Get content to show
+  private _getStepContentToShow(): IWizardStepProps {
+    const { steps } = this.props;
+
+    let stepToShow: IWizardStepProps | undefined = steps.find((wizStep: IWizardStepProps) => {
```

Comment:
>undefined [](start = 39, length = 9)

this should not be now right because we want to throw an error if it is undefined

Reply:
We throw error if there are no "steps".  This handles the case when there are steps, but none of them is marked as "Current".  In that case, I am returning the first step.

Reply:
I would say maybe throw an error instead of going to first step.

In general in cases that should never happen, it is always better to give an error instead of having a not-so-obvious-what-went-wrong behavior.